includes/features/stanford_jumpstart_home/homepage-mayfield.feature

```diff
@@ -66,17 +69,45 @@ Feature: Mayfield Homepage Layout
     | More about us                 | Content 3 column flow |
     | More information              | Content 3 column flow |
     | Learn more about our programs | Content 3 column flow |
-    | Facebook                      | Footer                |
-    | Twitter                       | Footer                |
-    | GooglePlus                    | Footer                |
-    | LinkedIn                      | Footer                |
-    | YouTube                       | Footer                |
-    | Vimeo                         | Footer                |
-    | Tumblr                        | Footer                |
-    | Pinterest                     | Footer                |
-    | Flickr                        | Footer                |
-    | sunetid@stanford.edu          | Footer                |
-    | Campus Map                    | Footer                |
-    | Stanford University           | Footer                |
-    | Research at Stanford          | Footer                |
-    | Stanford News                 | Footer                |
+
+  @api
+  Scenario: Set the homepage to Mayfield news and events
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/stanford-jumpstart/customize-design"
+    Then I press the "edit-layouts-stanford-jumpstart-home-mayfield-news-events-selector" button
```

Comment:
Do you want a step in here to select a different one first, in case this is selected already? (Like on the other Scenarios)